src/app/components/Footer/index.jsx

```diff
@@ -1,27 +1,119 @@
 import React from 'react';
 import styled from 'styled-components';
 import {
-  C_STORM,
+  C_GREY,
   C_WHITE,
   FF_NEWS_SANS_REG,
+  GEL_SPACING,
   GEL_SPACING_DBL,
 } from '../../lib/constants/styles';
 
 const StyledFooter = styled.footer`
-  background-color: ${C_STORM};
+  background-color: ${C_GREY};
   font-family: ${FF_NEWS_SANS_REG};
-  padding: ${GEL_SPACING_DBL}px;
+  padding: ${GEL_SPACING_DBL}px ${GEL_SPACING}px;
+`;
+
+const StyledList = styled.ul`
+  border-bottom: 1px solid ${C_WHITE};
+  display: grid;
+  grid-auto-flow: column;
+  grid-template-columns: repeat(2, 50%);
+  grid-template-rows: repeat(4, 50%);
+  list-style-type: none;
+  margin: 0 0 ${GEL_SPACING}px 0;
+  padding: 0 0 ${GEL_SPACING}px 0;
+  > li:first-child {
+    border-bottom: 1px solid ${C_WHITE};
+    grid-column: 1/3;
+    margin-bottom: ${GEL_SPACING}px;
+    @supports not (display: grid) {
+      width: 100%;
+    }
+  }
+`;
+
+const StyledListItem = styled.li`
+  min-width: 50%;
+  padding: ${GEL_SPACING}px;
+  @supports not (display: grid) {
+    padding: ${GEL_SPACING}px 0;
+    display: inline-block;
+  }
+`;
+
+const StyledLink = styled.a`
+  color: ${C_WHITE};
+  text-decoration: none;
+
+  &:hover,
+  &focus {
+    text-decoration: underline;
+  }
 `;
 
 const StyledParagraph = styled.p`
   color: ${C_WHITE};
+  padding: ${GEL_SPACING}px 0 0 0;
+  margin: 0;
 `;
 
+const listItems = [
+  {
+    href: 'https://www.bbc.com/news/help-41670342',
+    id: '1',
+    text: 'Why you can trust the BBC',
+  },
+  {
+    href: 'http://www.bbc.com/terms',
+    id: '2',
+    text: 'Terms of Use',
+  },
+  {
+    href: 'https://www.bbc.com/news/help-41670342',
+    id: '3',
+    text: 'About the BBC',
+  },
+  {
+    href: 'https://www.bbc.com/news/help-41670342',
+    id: '4',
+    text: 'Privacy Policy',
+  },
+  {
+    href: 'http://www.bbc.com/terms',
+    id: '5',
+    text: 'Cookies',
+  },
+  {
+    href: 'https://www.bbc.com/news/help-41670342',
+    id: '6',
+    text: 'Accessibility Help',
+  },
+  {
+    href: 'https://www.bbc.com/news/help-41670342',
+    id: '7',
+    text: 'Contact the BBC',
+  },
+];
+
+const externalLink = {
+  href: 'http://www.bbc.co.uk/help/web/links/',
+  text: 'Read about our approach to external linking',
+};
+
 const Footer = () => (
   <StyledFooter role="contentinfo">
+    <StyledList>
+      {listItems.map(listItem => (
+        <StyledListItem key={listItem.id}>
```

Comment:
The reason I did not add aria roles to the `ul` and `li` elements, as per @greenc05's suggestion, was that the jsx-a11y eslint plugin informed me that I should not include them. 
```
  22:5  error  The element ul has an implicit role of list. Defining this explicitly is redundant and should be avoided      jsx-a11y/no-redundant-roles
  23:7  error  The element li has an implicit role of listitem. Defining this explicitly is redundant and should be avoided  jsx-a11y/no-redundant-roles
```

Reply:
@sareh can we please remove that rule from jsx-a11y